exchanges/kucoin/kucoin_websocket.go

```diff
@@ -359,10 +359,13 @@ func (ku *Kucoin) processFuturesAccountBalanceEvent(respData []byte) error {
 		return err
 	}
 	ku.Websocket.DataHandler <- account.Change{
```

Comment:
This account change wont get processed as its passed to the websocket routine handler in `websocketroutine_manager.go` in function `websocketDataHandler`:
<img width="368" alt="image" src="https://github.com/user-attachments/assets/b862edc5-943d-409c-ba3f-fb43a3c1ae1d" />

Either we centralise everything to there and have it the main processor or we have to update this channel comms to `account.ProcessChange(g.Name, []account.Changes{changes}, creds)`

If we do it central to `websocketDataHandler` then all exchanges will probably benefit from the update with minimal changes, downside to that would be testing exchange side without an appropriate handler set up to update the actual accounts system. 

Let me know how you want to proceed.